pkg/cmd/codespace/common.go

```diff
@@ -104,6 +105,7 @@ type apiClient interface {
 	ListDevContainers(ctx context.Context, repoID int, branch string, limit int) (devcontainers []api.DevContainerEntry, err error)
 	GetCodespaceRepoSuggestions(ctx context.Context, partialSearch string, params api.RepoSearchParameters) ([]string, error)
 	GetCodespaceBillableOwner(ctx context.Context, nwo string) (*api.User, error)
+	GetHttpClient() (*http.Client, error)
```

Comment:
All the other `Get` methods on this interface correlate to an actual `GET` HTTP request, wondering if this makes more sense as:
```suggestion
	HTTPClient() (*http.Client, error)
```